### PR TITLE
Add missing changelog entries

### PR DESCRIPTION
**Pull request by CGNonofr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## [0.33.0]
 
+- The first parameter of all `monaco.languages.register*Provider` functions have changed to take a `DocumentSelector` instead of a single `languageId`
+- The `Environment.getWorker` function can now return a `Promise`
```
Comment (hediet) [inline CHANGELOG.md:6]: has changed ;)

**Comment by hediet** [comment]
Thank you!

File: CHANGELOG.md (hunk 1 above)
Comment (CGNonofr) [reply CHANGELOG.md]: I have no idea what you are talking about :disappointed_relieved: :smile: 

(Shame on me)

Comment (CGNonofr) [reply CHANGELOG.md]: (amended)

**Comment by CGNonofr** [comment]
Is there anything missing to get this approved?

**Pull request merged**